code-challenges/challenges-06.js

```diff
@@ -0,0 +1,81 @@
+'use strict';
+
+// -------------------------------------------------------------------------------------------------------
+//  Challenge 01:
+//  Required:
+//
+//  Write a function that takes an array and returns the maximum value in the provided array
+//  without using build-in functions.
+//  
+// Input: [12, 32, 22, 45, 78, 12, 50]
+// Output: 78
+//  
+
+const findMax = (arr)=>{
+    let max = arr[0];
+for(let i = 0 ; i < arr.length ; i++)
```

Comment:
Great work Ahmad
one note: inside the for loop, no need for i to start from 0 as you already assigned arr[0] to the max variable